client/src/Components/Forms.js

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from 'react';
+import SignInForm from './SignInForm';
+import SignUpForm from './SignUpForm';
+import './form.css';
+
+
+export default function Forms({isActive}) {
+  const [users, setUsers] = useState([]);
+  const [error, setError] = useState();
+  const [message, setMessage] = useState('');
+  const [status, setStatus] = useState('login');
+
+
+  useEffect(() => {
+    async function getUsers() {
+      try {
+        const res = await fetch('/api/Users');
+        if (!res.ok) {
+          throw new Error(`error ${res.status}`);
+        } else {
+        const jsonData = await res.json();
+        setUsers(jsonData);
+        }
+      } catch (err) {
+        setError(err);
+      }
+    }
+    getUsers();
+  }, []);
+
+
+  async function addUsers(newUser) {
+    try {
+      const res = await fetch('/api/Users/sign-up', {
+        method: "Post",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(newUser),
+      })
+      if (!res.ok) {
+        throw new Error(`error ${res.status}`);
+      } else {
+      const jsonData = await res.json();
+      if (jsonData.message) {
+        setMessage(jsonData.message);
+      } else {
+        setUsers(prev => users.concat(jsonData));
+        setMessage('Registration successful');
+      }
+    }
+    } catch (err) {
+      setError(err)
+    }
+  }
+
+  async function verifyUser(user) {
+    try {
+      const res = await fetch('/api/Users/sign-in', {
+        method: "Post",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(user),
+      })
+      if (!res.ok) {
+        throw new Error(`error ${res.status}`);
+      } else {
+      const jsonData = await res.json();
+      if (jsonData.message === 'Invalid Login') {
```

Comment:
Is it always going to be `'Invalid Login'`? Maybe you can just check if `jsonData.message` exists?